### PR TITLE
Indicate whether or not last message is yours

### PR DESCRIPTION
**Pull request by ghost**
This should be a fix for #1363 (plus I fixed a typo)
Sorry if the PR looks like a mess, it's my first attempt to use github. Hope it's fine

### CODE REVIEW

**Comment by ChaoticMind** [comment]
Try out `git rebase -i 586b743`, it lets you squash your commits and the PR will have a cleaner history.

**Comment by volcano99** [comment]
Thanks! I think it worked.

File: src/org/thoughtcrime/securesms/database/model/ThreadRecord.java [hunk 1]
```diff
@@ -78,9 +78,11 @@ public SpannableString getDisplayBody() {
     } else {
       if (TextUtils.isEmpty(getBody().getBody())) {
         return new SpannableString(context.getString(R.string.MessageNotifier_no_subject));
+      } else if (SmsDatabase.Types.isOutgoingMessageType(type)) {
+        return new SpannableString(context.getString(R.string.ThreadRecord_sent) + " " + getBody().getBody());
       } else {
-        return new SpannableString(getBody().getBody());
-      }
+            return new SpannableString(getBody().getBody());
```
Comment (chr1shaefn3r) [inline src/org/thoughtcrime/securesms/database/model/ThreadRecord.java]: Wrong intentation level.

**Comment by chr1shaefn3r** [comment]
I would favour a solution where I would also have the "delivered" information in the contact list. So additionally to the information that I send the last message, I would also know that it was delivered to the recipient.

**Comment by agrajaghh** [comment]
I agree, delived would be helpfull. What about some icons, maybe on the right side below the date?
checkmark for deliverd, arrow (to the right?) for sent messages and perhaps arrow to the left for received messages?

**Comment by volcano99** [comment]
Looks like that would require a little more time to develop, and considering I don't have the icons... anyone is welcome to trash my PR and open another one :-)

**Comment by agrajaghh** [comment]
@volcano99 I could assist you with the icon's if thats helping you?

**Comment by volcano99** [comment]
My main problem is that I don't have a ton of experience coding, so I'm afraid I'd end up slowing things down :/
Plus, first I want to hear from the official developers if they'd like the idea. Do you already know of any icons?

**Comment by agrajaghh** [comment]
I don't think its a really urgent issue, and the best way of gaining experience is just to try ;)

But you're right lets wait what the main devs will say...

Just made a quick mockup, but I'm not totally pleased with it though....
I tried the TS checkmark and a normal one .. I think the simple one looks better.

**Comment by volcano99** [comment]
The normal one looks SO much better, why is it not being used in conversations too?
(Just a thought... we need a pair of icons for each type, because they will change depending on the theme...)

**Comment by rhodey** [comment]
Can one of the admins verify this patch?

**Comment by mcginty** [comment]
Closing since this account was deleted. I'm in favor of displaying the delivery status of a message instead of a "You: " in the summary, since it provides even more information and still lets the user know it was a message sent from them.

**Pull request closed without merging**